src/DataLib/fendl32B_retrofit/process_fendl3.2.py

```diff
@@ -2,31 +2,44 @@
 import reaction_data as rxd
 import tendl_processing as tp
 import groupr_tools
+import file_handling as fh
+from pandas import concat
 
 def main():
     """
     Main method when run as a command line script.
     """
 
+    TAPE20 = 'tape20'
+    TAPE21 = 'tape21'
+
     mt_dict = rxd.process_mt_data(rxd.load_mt_table('mt_table.csv'))
+    cumulative_data = tp.cumulative_data
 
-    endf_path = 'tape20'
-    pendf_path = 'tape21'
+    for isotope, file_properties in fh.search_for_files().items():
+        element = file_properties['Element']
+        A = file_properties['Mass Number']
+        endf_path, pendf_path = file_properties['File Paths']
+        endf_path.rename(TAPE20)
+        pendf_path.rename(TAPE21)
```

Comment:
won't this result in gradually overwriting all the files in the directory?  Don't we want to copy the files to these names for processing?